frontend/packages/dev-console/src/components/health-checks/__tests__/health-check-probe-validation-utils.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { cloneDeep } from 'lodash';
+import { healthChecksValidationSchema } from '../health-check-probe-validation-utils';
+import { healthChecksDefaultValues } from '../health-check-probe-utils';
+
+export const mockHealthCheckFormData = {
```

Comment:
nit: can avoid export as used in this file

Reply:
removed export.